Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,6 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner =()=> <img src='https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png' alt='react icon' className='banner'></img>
```

Comment:
Good job using implicit return but remember when you have a single linke jsx component with nothing in between you can make it self closing like this `<img src="" />`